### PR TITLE
Add few badges

### PR DESCRIPTION
**Pull request by narendravaghela**
Added license, number of downloads and latest release badges in the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Lumen GraphQL
 
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/digiaonline/lumen-graphql/master/LICENSE)
```
Comment (hugovk) [inline README.md]: Let's just label it "License".

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+[![Total Downloads](https://poser.pugx.org/digiaonline/lumen-graphql/downloads)](https://packagist.org/packages/digiaonline/lumen-graphql)
+[![Latest Stable Version](https://poser.pugx.org/digiaonline/lumen-graphql/v/stable)](https://packagist.org/packages/digiaonline/lumen-graphql)
```
Comment (hugovk) [inline README.md]: Let's follow the same order as other projects, for example:
https://github.com/digiaonline/lumen-elasticsearch

**Review by hugovk — CHANGES_REQUESTED** [review]
Thanks! 

Please can you also add the Gitter badge from https://github.com/digiaonline/lumen-elasticsearch/blob/develop/README.md?

I don't think any of those other ones apply here yet, but if they do, please add those too.

**Comment by narendravaghela** [comment]
Sure, will do the changes.

**Comment by narendravaghela** [comment]
Added the Gitter badge and updated the display order. Other badges are related to CI, so we can add them after setting up CI on this project.

File: README.md (hunk 2 above)
Comment (narendravaghela) [reply README.md]: Done.

File: README.md (hunk 1 above)
Comment (narendravaghela) [reply README.md]: Done.

**Review by hugovk — APPROVED** [review]

**Comment by hugovk** [comment]
Thanks!

**Pull request merged**